### ISSUE TITLE
Notes on updating example-aster_stereo_reconstruction.ipynb 

### ISSUE THREAD

**Issue by dshean** [uw-cryo/asp_tutorials#9]
Just running this as a test and noticed it was still calling `stereo` for the ASTER example - we should update to `parallel_stereo`

Also, might be good to give people approximate runtimes in the doc above the cells running stereo/parallel_stereo, so they know how long to wait.  I was seeing ~2.5-3 m for all stereo runs (regardless of camera model type) on codespaces.

Should be clear that these are images from the ASTER VNIR instrument.

Maybe we can include the new `--aster-use-csm` option (https://stereopipeline.readthedocs.io/en/latest/examples/aster.html#using-the-csm-model).  Possibly even `jitter_solve`?  

It could also make sense to keep the intro/example tutorial relatively simple, and have a second notebook for the more advanced processing, potentially with a difference map for the output DEMs from different camera models.

There are some other typos, but can review later.  No rush on this, just wanted to jot down some notes.

**Comment by ShashankBice** [comment]
Agreed, parallel_stereo will be added soon when i add examples for running correlation using different schemes.

I would like to add use csm and jitter_solve in round 2, as advanced usage, when the first round of entry level examples are complete :)

**Comment by ShashankBice** [comment]
will add note on run-times for cells, great idea!
